packages/yoastseo/src/scoring/assessments/readability/SubheadingDistributionTooLongAssessment.js

```diff
@@ -199,10 +210,10 @@ class SubheadingsDistributionTooLong extends Assessment {
 	/**
 	 * Creates a marker for each subheading that precedes a text that is too long.
 	 *
-	 * @returns {Array} All markers for the current text.
+	 * @returns {Mark[]} All markers for the current text.
 	 */
 	getMarks() {
-		const marks = map( this.getTooLongSubheadingTexts(), function( { subheading } ) {
+		const marks = this.getTooLongSubheadingTexts().map( ( { subheading } ) => {
```

Comment:
Use native `map` here instead of lodash's map. According to [this page](https://www.measurethat.net/Benchmarks/Show/10250/0/lodash-map-vs-native-map-v2#:~:text=In%20summary%2C%20this%20benchmark%20compares,use%20case%20and%20personal%20preference.), the native `map` is faster.